Shared/src/shared/models/services/WebApiRequest.java

```diff
@@ -0,0 +1,4 @@
+package shared.models.services;
+
+public interface WebApiRequest {
```

Comment:
Is there much point in adding empty interfaces?   It just makes the code messy until someone gets round to doing something with them.

Reply:
It is empty on purpose and will probably stay this way indefinitely. This is what ApiClientService accepts to send over.